config/runtime.exs

```diff
@@ -55,12 +55,7 @@ if config_env() == :prod do
     client_id: spotify_client_id,
     client_secret: spotify_client_secret
 
-  database_path =
-    System.get_env("DATABASE_PATH") ||
-      raise """
-      environment variable DATABASE_PATH is missing.
-      For example: /etc/karaokium/karaokium.db
-      """
+  database_path = "/app/priv/repo/databases/karaokium_prod.db"
```

Comment:
If this is a fixed variable, it should be moved to the compile time config `prod.exs`. Only keep dynamic variables in `runtime.exs`.